core/modules/views/includes/view.inc

```diff
@@ -144,7 +144,7 @@ class view {
   // To avoid recursion in views embedded into areas.
   var $parent_views = array();
 
-  // Is the current stored view runned as an attachment to another view.
+  // Is the current stored view run as an attachment to another view.
```

Comment:
```suggestion
  // Is the current stored view run as an attachment to another view?
```

When phrased as a question (like this), it should end in a question mark.

Reply:
@bugfolder @BWPanda should we use "Whether ..." instead of asking a question here?

Also, isn't the past tense of "run" "ran"? ..although I'd prefer to use "added" or "loaded" instead, since views don't "run" - the queries that are generated by Views are executed. Right?

Reply:
In fact, "run" is unneeded:

```
// Is the current stored view an attachment to another view?
```

But yes, "Whether" is generally better than a question, so:

```
  // Whether the current stored view is an attachment to another view.
```

Reply:
> Also, isn't the past tense of "run" "ran"?

It is the Past Participle that is used as adjective, in this case _run_.

*Whether the current stored view is an attachment to another view.* makes sense for me, together *Whether the current stored view is used as attachment to another view.* (Maybe it is *as attachment for another view*?)